gnc/sub8_thruster_mapper/nodes/mapper.py

```diff
@@ -82,6 +85,10 @@ def get_thruster_wrench(self, position, direction):
         wrench_column = np.hstack([forces, torques])
         return np.transpose(wrench_column)
 
+    def get_b_matrix(self, srv):
```

Comment:
In the event of a thruster drop, the thruster mapper responds to an alarm to automatically generate a new B - do you want to handle updating B for the TGEN in the same way?